src/brex_elixirpb.proto

```diff
@@ -32,6 +32,15 @@ message EnumOptions {
   optional bool deprefix = 3;
 }
 
+message MessageOptions {
```

Comment:
nit: can we define this outside of the compiler in another proto inside credit card? Keeps brex logic outside of the compiler. 

Instead can you make a message option option in one of the tests protos to check if this works.

Reply:
wow it works!

Reply:
wdyt about define a general `MessgaeOption` in `brex_elixirpb.proto` file:
```
extend google.protobuf.MessageOptions {
  optional string message = 65009;
}
```

Then, for our own event message protos, we give option with a encoded map:
```
message MyEventMessage {
  option (brex.elixirpb.message) = "{\"is_event\":true}";
  google.protobuf.DoubleValue f1 = 1 [(brex.elixirpb.field).extype="float"];
}
```